app/controllers/Healthcheck.scala

```diff
@@ -1,15 +1,22 @@
 package controllers
 
+import com.typesafe.scalalogging.LazyLogging
 import play.api.mvc._
 import services.TouchpointBackend
+import play.api.libs.concurrent.Execution.Implicits._
 
-object Healthcheck extends Controller {
+object Healthcheck extends Controller with LazyLogging{
 
-  def index = Action {
-    Cached(1) {
-      if (TouchpointBackend.Normal.zuoraService.products.nonEmpty) Ok("OK")
-      else ServiceUnavailable("Service Unavailable")
-    }
+  def index = Action.async {
+      (for {
+        products <- TouchpointBackend.Normal.zuoraService.products if products.nonEmpty
+        sfAuth <- TouchpointBackend.Normal.salesforceService.repo.salesforce.getAuthentication
+      } yield Ok("OK"))
+      .recover { case t: Throwable =>
+        logger.error("Health check failed", t)
```

Comment:
Again minor, but we should use WARN log level at most, we don't want to raise alerts for that